comments/wagtail_hooks.py

```diff
@@ -25,6 +27,19 @@ class XtdCommentAdmin(ModelAdmin):
     button_helper_class = XtdCommentAdminButtonHelper
     menu_order = 601
 
+    def get_permissions_for_registration(self):
+        content_type = ContentType.objects.get_for_model(self.model)
+        Permission.objects.get_or_create(
+            codename='can_moderate_on_admin_panel',
+            content_type=content_type,
+            defaults={
+                'name': 'Can moderate comments on ADMIN PANEL',
+            }
+        )
+        permissions = super().get_permissions_for_registration()
+
+        return permissions.exclude(codename='can_moderate')
```

Comment:
In addition, there are other places where the 'can_moderate' permission is still used e.g. in templates.

Reply:
Would something like this work?

```python
p = Permission.objects.get(
    codename='can_moderate',
    content_type=ContentType.objects.get_for_model(self.model),
)
if p.name != self.CAN_MODERATE_LABEL:
    p.name = self.CAN_MODERATE_LABEL
    p.save()

return super().get_permissions_for_registration()
```

This might be preferable because then we won't have a 'hidden' permission, that existing users or groups may have, and that might take effect in unexpected ways when we upgrade the underlying comments app/library.